iiod/init/iiod.service.cmakein

```diff
@@ -1,14 +1,17 @@
+# SPDX-License-Identifier: LGPL-2.1+
+#
 # iiod - Systemd init script
 #
-# Copyright (C) 2016 Analog Devices Inc.
+# Copyright (C) 2016-2021 Analog Devices Inc.
```

Comment:
The above two options were sneaked in this patch...

Reply:
ACK